core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -419,9 +421,9 @@ protected Logger logger() {
     protected Path createVolumeDirectory(boolean temporary) {
         Path directory = new File(".tmp-volume-" + System.currentTimeMillis()).toPath();
         PathUtils.mkdirp(directory);
-
+        recursiveDeleteDir(Paths.get("/tmp/maven-hack"));
```

Comment:
FYI this method (`createVolumeDirectory`) is unused and to be deprecated, so I would not change it